matea-pjanic/assignments/liferay-app/sdk/portlets/music-portlet/docroot/view.jsp

```diff
@@ -45,145 +45,44 @@
 		</h3>
 
 		<div class="tracks-inner">
-			<div class="artist">
-				<div class="adele artist-photo">
-					<img class="playbutton" src="<%= PortalUtil.getPathContext(request) %>/images/play.png">
-				</div>
-
-				<div class="artist-name">
-					<liferay-ui:message key="adele" />
-				</div>
-
-				<div class="artist-song">
-					<liferay-ui:message key="send-my-love" />
-				</div>
-			</div>
-
-			<div class="artist">
-				<div class="artist-photo chainsmokers">
-					<img class="playbutton" src="<%= PortalUtil.getPathContext(request) %>/images/play.png">
-				</div>
-
-				<div class="artist-name">
-					<liferay-ui:message key="chainsmokers" />
-				</div>
-
-				<div class="artist-song">
-					<liferay-ui:message key="something-just-like-this" />
-				</div>
-			</div>
-
-			<div class="artist">
-				<div class="artist-photo deadmausphoto">
-					<img class="playbutton" src="<%= PortalUtil.getPathContext(request) %>/images/play.png">
-				</div>
-
-				<div class="artist-name">
-					<liferay-ui:message key="deadmaus" />
-				</div>
-
-				<div class="artist-song">
-					<liferay-ui:message key="saved" />
-				</div>
-			</div>
-
-			<div class="artist">
-				<div class="artist-photo kodakblackphoto">
-					<img class="playbutton" src="<%= PortalUtil.getPathContext(request) %>/images/play.png">
-				</div>
-
-				<div class="artist-name">
-					<liferay-ui:message key="kodak-black" />
-				</div>
-
-				<div class="artist-song">
-					<liferay-ui:message key="tunnel-vision" />
-				</div>
-			</div>
-
-			<div class="artist">
-				<div class="artist-photo chrisphoto">
-					<img class="playbutton" src="<%= PortalUtil.getPathContext(request) %>/images/play.png">
-				</div>
 
-				<div class="artist-name">
-					<liferay-ui:message key="chris-stapleton" />
-				</div>
+			<%
+				InputStream is_tracks = getServletContext().getResourceAsStream("/json/bends.json");
```

Comment:
Abstract logic for JSON reading to a utility class, for example
```
List<Bend> bends = MusicUtil.getBends();
List<Event> events = MusicUtil.getEvents();
````
MusicUtil should have a static constructor that reads files from disc.

MusicUtil should probably have a private helper method for reading a file.